app/controllers/membership_applications_controller.rb

```diff
@@ -68,6 +68,11 @@ def information
 
   end
 
+  def destroy
+    @membership_application.destroy
+    redirect_to membership_applications_url, notice: t('membership_applications.application_deleted')
```

Comment:
I noticed that the english wording (in yml file) for this is: ```Application deleted.```, yet the Swedish is ```Ansökan raderad.``` - which is plural ("Applications deleted.") and should be singular.

May want to fix in this PR, as it affects new tests in this PR.